chatkit-core/src/main/kotlin/com/pusher/chatkit/users/api/UserApiTypeMapper.kt

```diff
@@ -0,0 +1,17 @@
+package com.pusher.chatkit.users.api
+
+import com.pusher.chatkit.users.state.UserInternalType
+import com.pusher.chatkit.util.DateApiTypeMapper
+
+internal class UserApiTypeMapper(private val dateApiTypeMapper: DateApiTypeMapper) {
+    fun toUserInternalType(user: UserApiType): UserInternalType {
+        return UserInternalType(
+            id = user.id,
+            createdAt = dateApiTypeMapper.mapToEpochTime(user.createdAt),
+            name = user.name,
+            avatarUrl = user.avatarUrl,
+            updatedAt = dateApiTypeMapper.mapToEpochTime(user.updatedAt),
```

Comment:
let's use the same order as in the constructor, that is moving `updatedAt` named param straight after `createdAt` (it's nice and feels more logical to have then next to each other)

Reply:
also named params are not necessary here, indeed they just add noise

Reply:
👍 agree on the parameters should be in the same order. I feel like the named parameters are valuable as it will make it easier to spot a potential mistake (which happened in the v1 of the sdk).

Reply:
ok that's a good point, thanks! probably then it makes sense then to use them if there are more parameters of the same type